demo/addons/godot-openxr/scenes/first_person_controller_vr.gd

```diff
@@ -59,7 +69,8 @@ func initialise() -> bool:
 
 func _get_xr_viewport() -> Viewport:
 	if viewport:
-		return get_node(viewport)
+		var vp : Viewport = get_node(viewport)
```

Comment:
This is just to make Godots type checking happy. This ensures the `Node` returned is properly cast to `Viewport` and then return. Without this Godot will warn about the missing cast.